frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -180,7 +180,7 @@ class TableInteractive extends Component {
 
     if (this.props.isEmbeddingSdk) {
       const rootElement = document.getElementById(
-        EMBEDDING_SDK_ROOT_ELEMENT_ID,
+        EMBEDDING_SDK_PORTAL_CONTAINER_ELEMENT_ID,
```

Comment:
I guess the intent is to have everything styled, so we can measure the the actual size of this fake table. But we're rendering table here, and it doesn't have anything to do with portal.

Would keep using `EMBEDDING_SDK_ROOT_ELEMENT_ID` work in this case?

Reply:
Yeah I think we need the styles and therefore what we currently call portal.
Technically if we're rendering stuff on that from another component it's a portal, although hand-made

Reply:
That's fair. Let's keep this.